extension/test/unit/create-session-request.handler.spec.js

```diff
@@ -45,43 +45,45 @@ describe('get-payment-methods::execute::', () => {
   })
 
   it('handlePayment should return the right actions', async () => {
-    const adyenGetPaymentResponse = {
+    const adyenGetSessionResponse = {
       paymentMethods: [
```

Comment:
I think the `paymentMethods` attribute is not included in response from /session endpoint.
https://docs.adyen.com/api-explorer/Checkout/latest/post/sessions

Perhaps you can make a CURL command to this adyen endpoint and verify it?

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/1050/commits/8312d730202018c80d967ac0b4c25dd7adbfe75f

Yes, you are right.